.appveyor.yml

```diff
@@ -1,6 +1,6 @@
 image: Visual Studio 2017
 environment:
-  timeout: 10000
+  timeout: 30000
```

Comment:
please also reduce this, if possible to 15s, that should be enough. there are some tests that deliberately go to timeout, and 20 seconds more on a lot of tests and builds is a lot in total.